src/js/fee.js

```diff
@@ -125,9 +124,15 @@
 
       if (content !== 'raw') {
         returnContent = returnContent.replace(/<p>(?:<br ?\/?>|\u00a0|\uFEFF| )*<\/p>/g, '<p>&nbsp;</p>');
-        returnContent = returnContent.replace( /<p>\[/g, '[');
-        returnContent = returnContent.replace( /\]<\/p>/g, ']');
-        returnContent = returnContent.replace( /<br \/>/g, '');
+        var isWpautopEnabled = wp.fee.isWpautopEnabled;
+        if( isWpautopEnabled && isWpautopEnabled !== '' ) {
+          returnContent = window.switchEditors.pre_wpautop(returnContent);
+        } else {
+          returnContent = returnContent.replace(/<p>\[/g, '[');
+          returnContent = returnContent.replace(/\]<\/p>/g, ']');
+          returnContent = returnContent.replace(/<br \/>/g, '');
+          returnContent = returnContent.replace(/<p>&nbsp;<\/p>/g, '<br />');
+        }
```

Comment:
Could we add a quick comment about Why we are doing this? 
`# If we are not using wpautop, then we should do some basic filtering to prevent escaping shortcodes and other spacing controls`